packs/spell-effects/spell-effect-community-repair.json

```
@@ -0,0 +1,100 @@
+{
+    "_id": "dU4viL9kh554TKeB",
+    "img": "icons/commodities/tech/cog-brass.webp",
+    "name": "Spell Effect: Community Repair",
+    "system": {
+        "description": {
+            "value": "<p><strong>Critical Success</strong> The public work is restored to its prior construction, and it takes on further qualities, encouraged by the thoughts of the community. For the next week, while a caster is within 60 feet of the site of the ritual, they receive a +1 status bonus to Will saves against emotion effects and a +10-foot status bonus to their Speeds.</p>\n<p><strong>Critical Failure</strong> The public work refuses to repair itself, and the memories of the community members are soured in response to the botched ritual. For the next week, while a caster is within 60 feet of the site of the ritual, they cannot receive any benefit from beneficial emotion effects, and they take a -10-foot status penalty to their Speeds as their movement slows to a morose crawl.</p>"
+        },
+        "duration": {
+            "expiry": "turn-start",
+            "sustained": false,
+            "unit": "days",
+            "value": 7
+        },
+        "level": {
+            "value": 1
+        },
+        "rules": [
+            {
+                "choices": [
+                    {
+                        "label": "PF2E.Check.Result.Degree.Check.criticalSuccess",
+                        "value": 10
+                    },
+                    {
+                        "label": "PF2E.Check.Result.Degree.Check.criticalFailure",
+                        "value": -10
+                    }
+                ],
+                "flag": "outcome",
+                "key": "ChoiceSet",
+                "prompt": "PF2E.SpecificRule.Prompt.DegreeOfSuccess",
+                "rollOption": "community-repair-speed"
+            },
+            {
+                "key": "FlatModifier",
+                "predicate": [
+                    "community-repair-proximity"
```

```suggestion
                    "terrain:community-repair"
```
We prefer this kind of roll option, so while we're at the condensing we may as well clean this up a bit.